packages/quantic/package.json

```diff
@@ -30,7 +30,8 @@
     "setup:examples": "../../node_modules/.bin/ts-node scripts/build/deploy-community.ts",
     "deploy:main": "../../node_modules/.bin/sfdx force:source:deploy -u LWC -p force-app/main",
     "deploy:examples": "../../node_modules/.bin/sfdx force:source:deploy -u LWC -p force-app/examples",
-    "build:doc": "mkdir -p docs/out; jsdoc -c jsdoc-config.json > docs/out/quantic-docs.json",
+    "prebuild:doc": "mkdirp docs/out",
```

Comment:
is this script a monorepo thing that will be used automatically or it's just a utility for local setup?

Reply:
https://www.npmjs.com/package/mkdirp